dev/WHAT_GOES_INTO_THE_NEXT_RELEASE.md

```diff
@@ -29,11 +29,11 @@
 - [When proposed PRs are rejected?](#when-proposed-prs-are-rejected)
 - [Why are provider changes not cherry-picked?](#why-are-provider-changes-not-cherry-picked)
 - [What's the purpose of patch-level releases?](#whats-the-purpose-of-patch-level-releases)
-- [Do we intentionally skip over some changes when releasing a patchlevel release?](#do-we-intentionally-skip-over-some-changes-when-releasing-a-patchlevel-release)
+- [Do we intentionally skip over some changes when releasing a patch level release?](#do-we-intentionally-skip-over-some-changes-when-releasing-a-patch-level-release)
```

Comment:
I think we should just refer to it as "patch release".

Reply:
Good point. Checked SemVer.org and also there it is just called "patch release", so removed all the terms of "level".